To: vim_dev@googlegroups.com
Subject: Patch 7.3.328
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.328
Problem:    When command line wraps the cursor may be displayed wrong when
	    there are multi-byte characters.
Solution:   Position the cursor before drawing the text. (Yasuhiro Matsumoto)
Files:	    src/ex_getln.c


*** ../vim-7.3.327/src/ex_getln.c	2011-09-30 16:23:21.000000000 +0200
--- src/ex_getln.c	2011-09-30 17:39:27.000000000 +0200
***************
*** 2870,2875 ****
--- 2870,2876 ----
  	{
  	    msg_no_more = TRUE;
  	    i = cmdline_row;
+ 	    cursorcmd();
  	    draw_cmdline(ccline.cmdpos, ccline.cmdlen - ccline.cmdpos);
  	    /* Avoid clearing the rest of the line too often. */
  	    if (cmdline_row != i || ccline.overstrike)
*** ../vim-7.3.327/src/version.c	2011-09-30 17:30:27.000000000 +0200
--- src/version.c	2011-09-30 17:45:28.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     328,
  /**/

-- 
To define recursion, we must first define recursion.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
